iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/sql/ast/Fill.java

```diff
@@ -165,4 +168,24 @@ public boolean shallowEquals(Node other) {
         && Objects.equals(timeColumnIndex, fill.timeColumnIndex)
         && Objects.equals(fillGroupingElements, fill.fillGroupingElements);
   }
+
+  @Override
+  public long ramBytesUsed() {
+    long size = INSTANCE_SIZE;
+    size += AstMemoryEstimationHelper.getEstimatedSizeOfNodeLocation(getLocationInternal());
+    if (fillValue != null) {
+      size += AstMemoryEstimationHelper.getEstimatedSizeOfAccountableObject(fillValue);
+    }
+    if (timeColumnIndex != null) {
+      size += AstMemoryEstimationHelper.getEstimatedSizeOfAccountableObject(timeColumnIndex);
+    }
+    if (fillGroupingElements != null) {
+      size += AstMemoryEstimationHelper.getEstimatedSizeOfNodeList(fillGroupingElements);
+    }
+    // TimeDuration is a simple object, estimate its size
+    if (timeBound != null) {
+      size += RamUsageEstimator.shallowSizeOfInstance(TimeDuration.class);
```

Comment:
```suggestion
      size += TimeDuration.INSTANCE_SIZE;
```
define INSTANCE_SIZE in TimeDuration class